.github/workflows/ci-all-in-one-build.yml

```diff
@@ -48,3 +48,4 @@ jobs:
       env:
         DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
         QUAY_TOKEN: ${{ secrets.QUAY_TOKEN }}
+        SKIP_DEBUG: ${{inputs.SKIP_DEBUG}}
```

Comment:
We don't run actions manually. The main release is done by `.github/workflows/ci-release.yml`, so there it should be enabling debug builds, and here they should be explicitly disabled.